src/common/mfu_daos.c

```diff
@@ -1313,3 +1315,120 @@ int mfu_flist_copy_daos(
 
     return rc;
 }
+
+int mfu_daos_hdf5_copy(mfu_file_t *mfu_src_file,
+                       mfu_file_t *mfu_dst_file,
+                       char **argpaths,
+                       daos_args_t *daos_args)
+{
+    int rc = 0;
+    int size;
+
+    MPI_Comm_size(MPI_COMM_WORLD, &size);
+    if (size > 1) {
+        MFU_LOG(MFU_LOG_ERR, "dcp uses h5repack to copy DAOS HDF5 containers "
+                "to and from a POSIX filesystem. Only one process "
+                "is currently supported, restart program using only "
+                "one MPI process "
+                MFU_ERRF, MFU_ERRP(-MFU_ERR_INVAL_ARG));
+                rc = 1;
+                return rc;
+    }
+
+    if (mfu_src_file->type == HDF5 || mfu_dst_file->type == HDF5) {
+        char src_cont_uuid[130];
+        char dst_cont_uuid[130];
+        pid_t child;
+        int child_status;
+        pid_t child_pid;
+        char *args[4];
+        args[0] = "h5repack";
```

Comment:
@dsikich After the serialization changes, we can use
```suggestion
        args[0] = HDF5_BIN_DIR"/h5repack";
```